postdeploy/scripts/certmanager-install.sh

```diff
@@ -1,11 +0,0 @@
-#!/bin/bash
```

Comment:
why has this file been removed?

Reply:
Great catch! Readded certmanager-install.sh as per offline discussion.